### PR TITLE
support for custom role to permissions mappings

### PR DESCRIPTION
**Pull request by mrmauer**
[BE PR](https://github.com/PropelAuth/auth/pull/615)
[Notion](https://www.notion.so/Role-Permissions-Mappings-269f8b5f346645fdb0fa8caa391d9c5d)

### CODE REVIEW

**Comment by mrmauer** [comment]
new features tested successfully w/ example app

File: src/apis/org_service_api.rs [hunk 1]
```diff
@@ -184,6 +200,16 @@ pub enum UpdateOrgError {
     UnknownValue(serde_json::Value),
 }
 
+/// struct for typed errors of method [`update_org`]
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(untagged)]
+pub enum SubscribeOrgToRoleMappingError {
+    Status400(crate::models::BadUpdateOrgRequest),
```
Comment (itailevi98) [inline src/apis/org_service_api.rs:207]: Should this be a `BadUpdateOrgRequest` error here?

**Review by itailevi98 — APPROVED** [review]
lgtm, just left one question

File: src/apis/org_service_api.rs (hunk 1 above)
Comment (mrmauer) [reply src/apis/org_service_api.rs:207]: hits the same route, to I used the same error type

**Pull request merged**